RELEASING.md

```diff
@@ -1,34 +1,74 @@
 # Releasing
 
-## Go SDK
+> **Warning**
+> Currently, if any SDK bumps an engine dependency, we should bump the engine dependency of all other SDKs concurrently. This helps ensure that users running multiple SDKs from a single machine all use the same engine. This is important as currently our automatic provisioning code enforces the existence of a single engine running at a time.
+
+## Engine
 
 ### Prep
 
+- Ensure you have GHCR credentials for your Dagger Github account configured ([instructions](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry)).
+- Ensure [Mage is installed](https://magefile.org/).
+- Ensure that CI is green for all tests in main
+
 ```console
 git pull
 git status  # make sure everything is clean
 ```
 
-### Release Engine
+### Bump version tags
+
+- Modify `EngineImageRef` in `internal/mage/engine.go` to use a tag set to the engine version (e.g. `v0.3.0`).
```

Comment:
I think this is automatable. Can have the code look at the current git branch and use that to set the tag. Will need to add some sort of confirmation dialog too. I think that slight extra bit of complication would be worth it to get this more automated, but let me know if you disagree

Reply:
Yes, that is exactly what @aluzzardi mentioned earlier today - `mage sdk:python:engine:bump` - so we are all on the same page.